LICENSE.md

```diff
@@ -0,0 +1,11 @@
+Copyright 2018 One Identity LLC.  ALL RIGHTS RESERVED.
```

Comment:
The pp.license file gets added to the top of the resulting script, that is why it was made up of shell-style comments.  If you get rid of pp.license, the Makefile needs to add the comment characters back before generating the pp script.

Reply:
makes sense.  I'll close the request and just generate a new license.md file.  Just doing some administrative house keeping.